paraview/xmls/MergeTreeAutoencoder.xml

```diff
@@ -0,0 +1,653 @@
+    
+      <InputProperty
+        name="Info"
+        port_index="2"
+        command="SetInputConnection">
+        <ProxyGroupDomain name="groups">
+          <Group name="sources"/>
+          <Group name="filters"/>
+        </ProxyGroupDomain>
+        <DataTypeDomain name="input_type">
+          <DataType value="vtkTable"/>
+        </DataTypeDomain>
+        <InputArrayDomain name="input_scalars" number_of_components="1">
+          <Property name="Input" function="FieldDataSelection" />
+        </InputArrayDomain>
+        <Documentation>
+          Additionnal information such as cluster assignment for cluster loss.
+        </Documentation>
+      </InputProperty>
+
+      <!-- INPUT PARAMETER WIDGETS -->       
+      <IntVectorProperty
+      name="NormalizedWasserstein"
+      command="SetNormalizedWasserstein"
+      label="Normalized Wasserstein"
+      number_of_elements="1"
+      default_values="1">
+        <Documentation>
+          Choose between the normalizd Wasserstein or the usual Wasserstein metric. If merge trees are provided in input, they will automatically be converted to persistence diagrams if this option is disabled.
+        </Documentation>
+        <BooleanDomain name="bool"/>
+      </IntVectorProperty>
+    
+      <IntVectorProperty
+      name="NumberOfEncoderLayers"
+      command="SetNumberOfEncoderLayers"
+      label="Number of Encoder Layers"
+      number_of_elements="1"
+      default_values="1">
+        <Documentation>
+          Number of layers in the encoder (without counting the latent space layer).
+        </Documentation>
+      </IntVectorProperty>
+      
+      <IntVectorProperty
+      name="ScaleLayerAfterLatent"
+      command="SetScaleLayerAfterLatent"
+      label="ScaleLayerAfterLatent"
+      number_of_elements="1"
+      default_values="0"
+      panel_visibility="advanced">
+        <Documentation>
+          By default, the latent layer and the layer after it will have the same number of axes. This option allows to set the number of axes in the layer after the latent one as the average with the layer after it.
+        </Documentation>
+        <BooleanDomain name="bool"/>
+      </IntVectorProperty>
+      
+      <IntVectorProperty
+      name="InputNumberOfAxes"
+      command="SetInputNumberOfAxes"
+      label="Input Number of Axes"
+      number_of_elements="1"
+      default_values="16"
+      panel_visibility="advanced">
+        <Documentation>
+          Number of axes in the input layer.
+        </Documentation>
+      </IntVectorProperty>
+      
+      <DoubleVectorProperty
+      name="InputOriginPrimeSizePercent"
+      command="SetInputOriginPrimeSizePercent"
+      label="Input Origin Prime Size Percent"
+      number_of_elements="1"
+      default_values="15"
+      panel_visibility="advanced">
+        <Documentation>
+          Adjust the maximum size of the origin of the first output basis, given the total number of nodes in the input.
```

Comment:
`given the total number of nodes in the input`
->
`as a percentage of the total number of nodes in the input` (?)

also, to discuss `CMake/merge_tree_preprocess.xml`:
- `Epsilon1UseFarthestSaddle`: missing documentation
- `DeleteMultiPersPairs`: to discuss